torch/csrc/PtrWrapper.cpp

```diff
@@ -44,7 +50,7 @@ static PyObject * THPWrapper_pynew(PyTypeObject *type, PyObject *args, PyObject
   return self;
 }
 
-static void THPWrapper_dealloc(THPWrapper* self)
+static void THPWrapper_dealloc(THPWrapper* self) __ubsan_ignore_function__
```

Comment:
This also looks like a bug.

Reply:
Noted in https://github.com/pytorch/pytorch/issues/9054

Reply:
Put a comment here, so that someone looking at this can easily figure it out :)